var/spack/repos/builtin/packages/sublime-text/package.py

```diff
@@ -33,8 +33,8 @@ class SublimeText(Package):
     homepage = "http://www.sublimetext.com/"
     url      = "https://download.sublimetext.com/sublime_text_3_build_3126_x64.tar.bz2"
 
-    version('3126',  'acc34252b0ea7dff1f581c5db1564dcb')
-    version('2.0.2', '699cd26d7fe0bada29eb1b2cd7b50e4b')
+    version('3_build_3126', 'acc34252b0ea7dff1f581c5db1564dcb')
```

Comment:
I actually added this package for fun one time. Sublime Text 2 was released a while ago. Sublime Text 3 does not yet have an official release, but they do provide beta builds.